Dockerfile

```
@@ -84,6 +84,7 @@ COPY apps/playlist apps/playlist
 COPY apps/investigations apps/investigations
 COPY apps/advisor apps/advisor
 COPY apps/dashboard apps/dashboard
+COPY pkg/infra/log/go.* pkg/infra/log
```

why not just copy the folder as it is?
```suggestion
COPY pkg/infra/log pkg/infra/log
```